bundler/lib/dependabot/bundler/package_manager.rb

```diff
@@ -9,9 +9,8 @@ module Dependabot
   module Bundler
     PACKAGE_MANAGER = "bundler"
 
-    SUPPORTED_BUNDLER_VERSIONS = T.let([
-      Version.new("2")
-    ].freeze, T::Array[Dependabot::Version])
+    # Keep versions in ascending order
+    SUPPORTED_BUNDLER_VERSIONS = T.let([Version.new("2")].freeze, T::Array[Dependabot::Version])
```

Comment:
Keeping the array in ascending order is important because we compare the first element of `supported_versions` with the current version (`version < supported_versions.first`) to determine if it is unsupported.

Reply:
We added specs to check if this array is ordered, so we don't need to run sorting.

Reply:
_Assuming_ that we'd only support versions greater than X, we could remove the requirement of having developers manually sorting by checking that a version is greater than all versions in this collection.

That would fall apart if we supported, say PackageManager `v1` and `v3`, but not `v2`. However, that would also fall apart here.

Reply:
applied the change to compare with each of them.